package.json

```diff
@@ -25,12 +25,12 @@
     "@types/node": "^12.7.12",
     "jest": "^24.9.0",
     "nock": "^11.4.0",
-    "ts-jest": "^24.1.0",
+    "ts-jest": "^26.4.4",
```

Comment:
I believe you should also be updating @types/jest to match.

Reply:
snowball down the hill scenario.  Updating @types/jest meant we needed to update @types/node and even bump up typescript so the thing would actually build.

Reply:
Fun times! 😀